tests/frontends/test_pytorch_ops.py

```diff
@@ -315,6 +270,9 @@ def _name_args_helper(name, args):
 # all_torch_ops = dir(torch)
 # all_torch_tensor_ops = dir(torch.Tensor([5.49670]))
 
+# pytest_timeout used so that unmapped ops with infinite call recursion
+# don't take very long time to fail
+pytest_timeout_time = 10
```

Comment:
#238 adds a 30s timeout for every test, so I would remove specific timeouts like the ones below.